cloudbuild.yaml

```diff
@@ -74,70 +83,80 @@ steps:
     docker build -t ${_REPOSITORY}/batch-pipeline:${_TAG} .;
     cd ../streaming-binlog;
     docker build -t ${_REPOSITORY}/streaming-pipeline:${_TAG} .;
+  waitFor: ['Compile Bunsen and Pipeline']
 
 - name: '${_REPOSITORY}/batch-pipeline:${_TAG}'
   id: 'Run Batch Pipeline in FHIR-search mode with HAPI source'
   env:
   - FHIR_SERVER_URL=http://hapi-server:8080/fhir
   - PARQUET_PATH=/workspace/e2e-tests/FHIR_SEARCH_HAPI
-  - SINK_PATH=http://sink-server:8080/fhir
+  - SINK_PATH=http://sink-server1:8080/fhir
   - SINK_USERNAME=hapi
   - SINK_PASSWORD=hapi
+  waitFor: ['Build Pipeline Images', 'Upload to HAPI']
 
 - name: '${_REPOSITORY}/e2e-tests:${_TAG}'
   id: 'Run E2E Test for FHIR-search mode with HAPI source'
   env:
   - PARQUET_SUBDIR=FHIR_SEARCH_HAPI
   - DOCKER_NETWORK=--use_docker_network
+  - SINK_SERVER=http://sink-server1:8080
+  waitFor: ['Run Batch Pipeline in FHIR-search mode with HAPI source']
```

Comment:
I suppose this should wait for `Build E2E Image` too?

Reply:
Yes right, I have included these changes in the latest commit. Update the flowchart as well.